blocks/faas-decode/faas-decode.js

```diff
@@ -6,8 +6,7 @@ export default async function init(el) {
   const resp = await fetch(url);
 
   if (!resp?.ok) {
-    // eslint-disable-next-line no-console
-    console.log(`Error fetching data from url: ${url}`);
+    window.lana?.log(`Error fetching data from url: ${url}`, { tags: 'info, faas-decode' });
```

Comment:
I remember the tags being encoded at one point, I'm not sure if the space after the comma yields the expected results. Was this tested with the stage logs?